main.go

```diff
@@ -62,11 +62,13 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
+	var (
+		enableLeaderElection bool
+		probeAddr            string
+		metricsAddr          string
+	)
 
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metric endpoint binds to.")
```

Comment:
This disables metrics for the operator. I wasn't 100% sure on doing it, but it seems like we either need to secure it or shut it off?

Reply:
I'm in favor of shutting it off like this.